### PR TITLE
Falls back to an avatar on S3 for enterprise slack avatars

### PR DESCRIPTION
**Pull request by discorick**
Addresses comments in https://github.com/huboard/huboard/issues/619

### CODE REVIEW

File: lib/hu_board/services/slack.rb [hunk 1]
```diff
@@ -26,10 +26,17 @@ def github_url
       ENV["GITHUB_WEB_ENDPOINT"] || "https://github.com/"
     end
 
+    def avatar_url(mash)
+      if github_url != 'https://github.com/'
```
Comment (discorick) [inline lib/hu_board/services/slack.rb]: @rauhryan @dahlbyk this is the best way I know to differentiate between enterprise and not on the fly, if theres a better key let me know.

Comment (dahlbyk) [inline lib/hu_board/services/slack.rb]: I assumed there would be an environment variable we could check...?

**Comment by dahlbyk** [comment]
Change looks good other than the question of identifying enterprise. I'm not a fan of the URL check, so if there isn't a clean way to identify Enterprise let's add one. (Even if that just means committing some sort of global `enterprise = true` change on the enterprise branch.)

**Comment by discorick** [comment]
Cool, I just need @rauhryan to glance at the ENV var and let me know if theres something better in place already.

**Pull request merged**